src/components/shared/CallNodeContextMenu.js

```diff
@@ -545,15 +545,13 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
 
         {showExpandAll ? (
           <>
-            <Localized id="CallNodeContextMenu--expand-all">
-              <MenuItemWithShortcut
-                onClick={this._handleClick}
-                data={{ type: 'expand-all' }}
-                shortcut="*"
-              >
-                Expand all
-              </MenuItemWithShortcut>
-            </Localized>
+            {this.renderMenuItemWithShortcut({
+              l10nId: 'CallNodeContextMenu--expand-all',
+              onClick: this._handleClick,
+              data: { type: 'expand-all' },
+              shortcut: '*',
+              content: 'Expand all',
```

Comment:
Yeah, I made another commit so it would be easier to review. But since you are done already (and there was a mistake anyway) I amended  it to the previous ones.